projects/playground/src/app/app.component.spec.ts

```diff
@@ -127,7 +118,7 @@ describe('AppComponent', () => {
       expect(authMock.logout).toHaveBeenCalledWith({
         localOnly: false,
         federated: false,
-        returnTo: 'http://localhost:4200',
+        returnTo: 'http://localhost:9876',
```

Comment:
I noticed this changed. Why is that? Does it make sense to try to fix it to the previous value?

Reply:
The actual value doesn't really matter, just that it matches what is provided by the window. Previously we were specifically mocking `WindowService`, but now it uses the built-in `Document` type which has a different value. It's not hugely important to the test, the key is that `returnTo` was specified.